interface-definitions/service_monitoring_prometheus.xml.in

```diff
@@ -36,6 +36,31 @@
                   #include <include/interface/vrf.xml.i>
                 </children>
               </node>
+              <node name="blackbox-exporter">
+                <properties>
+                  <help>Prometheus exporter for probing endpoints</help>
+                </properties>
+                <children>
+                  #include <include/listen-address.xml.i>
+                  #include <include/port-number.xml.i>
+                  <leafNode name="port">
+                    <defaultValue>9115</defaultValue>
+                  </leafNode>
+                  #include <include/interface/vrf.xml.i>
+                  <leafNode name="config-file">
```

Comment:
I am no fan at all of out-of CLI configuration files / snippets. Those usually do more harm then good.
We should rather add the required CLI knows instead.

Reply:
I've added basic config generation for dns/icmp which are most likely used for testing upstream connectivity